docs/BR.md

```diff
@@ -2887,7 +2910,18 @@ This section contains several fields that are common among multiple certificate
 
 ##### 7.1.2.11.2 CRL Distribution Points
 
-If present, the CRL Distribution Points extension MUST contain at least one `DistributionPoint`; containing more than one is NOT RECOMMENDED. All `DistributionPoint` items must be formatted as follows:
+The CRL Distribution Points extension MUST be present in:
+- Subordinate CA Certificates; and
+- Subscriber Certificates that 1) do not qualify as "Short-lived Subscriber Certificates" and 2) do not include an Authority Information Access extension with an id-ad-ocsp accessMethod.
+
+The CRL Distribution Points extension SHOULD NOT be present in:
+- Root CA Certificates; and
+- Short-lived Subscriber Certificates.
```

Comment:
I think "SHOULD NOT be present in...Short-lived Subscriber Certificates" is a step too far.

CAs that issue <=398-day certificates may sometimes issue <10-day certificates, but not because they want to avoid having to provide revocation information.  For example: if a Subscriber suffers a key compromise 7 days before their certificate is due to expire, the CA might issue a 7-day replacement certificate (with a different public key) that will expire at the same time, in order to cover (but not exceed) the validity period that the Subscriber had already paid for.

```suggestion
The CRL Distribution Points extension is OPTIONAL in:
- Short-lived Subscriber Certificates.
```

Reply:
No concerns with this recommendation. 

Addressed in [this](https://github.com/ryancdickson/staging/pull/5/files) PR.